example.js

```diff
@@ -9,28 +9,34 @@ rosnodejs.initNode('/my_node')
   // EXP 1) Service Server
   let service = rosNode.advertiseService('/set_bool', SetBool,
     (req, resp) => {
-      console.log('Handling request! ' + JSON.stringify(req));
+      rosnodejs.log.info('Handling request! ' + JSON.stringify(req));
```

Comment:
I think we could provide that - `rospy` provides all the methods with a `log` prefix to them (`logerr`, `loginfo`, `logwarn`, etc). This method lets you pull the logger out yourself and never refer to `rosnodejs` again though.  So you could do
```
let log = rosnodejs.log;
log.info('A Message!');
```
You can also generate your owned named logger as a child of the root logger if you want to (and then it would be available through `/get_loggers` and  `/set_logger_level`)
```
let customLogger = rosnodejs.log.getLogger('MyLogger');
customLogger.info('My other message');
```
You're less likely to get  `throttle` or `once` clashes too if you create your own loggers as they each keep their own sets.
The `log` property from `rosnodejs` is actually the `LoggingManager` though.  It exposes the logging methods of its root logger for ease of use. So all the methods available through `rosnodejs.log` are not necessarily available through loggers that you ask it to create.